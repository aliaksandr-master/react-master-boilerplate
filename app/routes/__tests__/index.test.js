/* eslint-env browser, jest */

import routesTree from '../index';



describe('routes', () => {
  it('should require without errors', () => {
    expect(routesTree.getRoutes()).toMatchSnapshot();
  });
});
