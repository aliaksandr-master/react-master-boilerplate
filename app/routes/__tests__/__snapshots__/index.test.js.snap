// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`routes should require without errors 1`] = `
Object {
  "app": Route {
    "additionalParams": Array [],
    "availableParams": Array [],
    "availableUrlParams": Array [],
    "name": "app",
    "params": Object {},
    "path": "/app",
    "pathMask": "/app",
    "pathMaskExp": /\\^\\\\/app\\$/,
    "pathMaskExpSrc": "/app",
    "pathMaskParams": Array [],
    "reroute": null,
    "searchParams": Array [],
    "slots": Object {
      "root": [Function],
    },
    "url": "/app",
  },
  "app.dashboard": Route {
    "additionalParams": Array [],
    "availableParams": Array [],
    "availableUrlParams": Array [],
    "name": "app.dashboard",
    "params": Object {},
    "path": "/app/dashboard",
    "pathMask": "/app/dashboard",
    "pathMaskExp": /\\^\\\\/app\\\\/dashboard\\$/,
    "pathMaskExpSrc": "/app/dashboard",
    "pathMaskParams": Array [],
    "reroute": null,
    "searchParams": Array [],
    "slots": Object {
      "appBody": [Function],
      "root": [Function],
    },
    "url": "/app/dashboard",
  },
  "home": Route {
    "additionalParams": Array [],
    "availableParams": Array [],
    "availableUrlParams": Array [],
    "name": "home",
    "params": Object {},
    "path": "/",
    "pathMask": "/",
    "pathMaskExp": /\\^\\\\/\\$/,
    "pathMaskExpSrc": "/",
    "pathMaskParams": Array [],
    "reroute": null,
    "searchParams": Array [],
    "slots": Object {
      "root": [Function],
    },
    "url": "/",
  },
  "login": Route {
    "additionalParams": Array [],
    "availableParams": Array [],
    "availableUrlParams": Array [],
    "name": "login",
    "params": Object {},
    "path": "/login",
    "pathMask": "/login",
    "pathMaskExp": /\\^\\\\/login\\$/,
    "pathMaskExpSrc": "/login",
    "pathMaskParams": Array [],
    "reroute": null,
    "searchParams": Array [],
    "slots": Object {
      "root": [Function],
    },
    "url": "/login",
  },
  "logout": Route {
    "additionalParams": Array [],
    "availableParams": Array [],
    "availableUrlParams": Array [],
    "name": "logout",
    "params": Object {},
    "path": "/logout",
    "pathMask": "/logout",
    "pathMaskExp": /\\^\\\\/logout\\$/,
    "pathMaskExpSrc": "/logout",
    "pathMaskParams": Array [],
    "reroute": null,
    "searchParams": Array [],
    "slots": Object {
      "root": [Function],
    },
    "url": "/logout",
  },
  "register": Route {
    "additionalParams": Array [],
    "availableParams": Array [],
    "availableUrlParams": Array [],
    "name": "register",
    "params": Object {},
    "path": "/register",
    "pathMask": "/register",
    "pathMaskExp": /\\^\\\\/register\\$/,
    "pathMaskExpSrc": "/register",
    "pathMaskParams": Array [],
    "reroute": null,
    "searchParams": Array [],
    "slots": Object {
      "root": [Function],
    },
    "url": "/register",
  },
}
`;
